Tests/scripts/update_id_set.py

```diff
@@ -305,12 +309,11 @@ def process_integration(file_path):
     """
     res = []
     if os.path.isfile(file_path):
-        if re.match(INTEGRATION_REGEX, file_path, re.IGNORECASE) or \
-                re.match(BETA_INTEGRATION_REGEX, file_path, re.IGNORECASE):
+        if checked_type(file_path, (INTEGRATION_REGEX, BETA_INTEGRATION_REGEX, PACKS_INTEGRATION_YML_REGEX)):
             print("adding {0} to id_set".format(file_path))
             res.append(get_integration_data(file_path))
-    else:  # In case we encountered a package
-        for yml_file in glob.glob(os.path.join(file_path, '*.yml')):
+    else:
+        for yml_file in glob.glob(os.path.join(file_path, os.path.basename(file_path) + '.yml')):
```

Comment:
no need to iterate, remove the for.
to be safe, add `os.path.isfile` to check that the file actually exists.

Reply:
you missed this one